src/components/ValidationMessage/ErrorIcon.tsx

```diff
@@ -0,0 +1,14 @@
+type Props = {}
+
+const ErrorIcon = ({}: Props) => {
```

Comment:
```suggestion
const ErrorIcon = () => {
```

If the component has no props, there is no need to declare them